_posts/2018-10-15-arms-proposal.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title:  "ARMS CSC 596 Proposal"
+categories: ["Missouri State University", "ARCS"]
+tags: ["self-study", "csc596", "robotics", "python"]
+author: "Dillon Flohr"
+---
+
+# CSC 596 Proposal
+
+### ARMS (Autonomous Robot Markup Syntax) for ARCS
+
+### Student: Dillon Flohr
+
+### Supervisor: Anthony J. Clark
+
+In robot simulation, there are a variety of formats used to specify the parts, dimensions, and features of a robot. It can be very laborious to repeatedly create these specifications for the same robot for different simulation systems, when the only difference could be the formatting of the XML input. The goal of this self-study course is to create a language and interpreter that will take a text file written in Autonomous Robot Markup Syntax (ARMS), a Light Markup Language we will create, and output various robot specification files in the form of code. These files will be usable in their respective simulation environment and all come from the same source file.
+
+We will be focusing on creating ARMS and getting it working for the Open Dynamics Engine. This alone will take up the entirety of the course work, and any extra time we may have after completing the first format will be used to start the other targets.
+
+We will be meeting weekly during the first half of the semester on Thursdays from 4:00 to 4:45. For the second half of the semester we will meet at this time bi-weekly. We will also be using the ARCS Slack channel to communicate.
```

Comment:
We will be meeting weekly during the first half of the semester --> During the first half of the semester, we will meet weekly